File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomPipeline/DbAtomPipelineStatusEnum.cs
namespace DataModelLibrary.Database.AtomPipeline;

/// <summary>資料庫-原子-商機-狀態-列舉</summary>
public enum DbAtomPipelineStatusEnum : short
{
    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>連字號</summary>
    Hyphen = 1,

    /// <summary>eDM已開啟</summary>
    Opened = 2,

    /// <summary>eDM已點擊</summary>
    Clicked = 3,

    /// <summary>已轉電銷</summary>
    TransferredToSales = 4,

    /// <summary>已完成電銷</summary>
    CompletedSales = 5,

    /// <summary>已轉業務</summary>
    TransferredToBusiness = 6,

    /// <summary>商機10%：業務接受 或 booking 接受</summary>
    Business10Percent = 7,

    /// <summary>商機30%</summary>
    Business30Percent = 8,

    /// <summary>商機70%</summary>
    Business70Percent = 9,

    /// <summary>商機100%</summary>
    Business100Percent = 10,

    /// <summary>商機失敗</summary>
    BusinessFailed = 11,

    /// <summary>已轉專案</summary>
    TransferredToProject = 12
}

File: FrontEnd/manager-back-site/src/constants/DbAtomPipelineStatusEnum.ts
/** 資料庫-原子-商機-狀態-列舉 */
export enum DbAtomPipelineStatusEnum {
  /** 未定義 */
  Undefined = 0,
  /** 連字號 */
  Hyphen = 1,
  /** eDM已開啟 */
  Opened = 2,
  /** eDM已點擊 */
  Clicked = 3,
  /** 已轉電銷 */
  TransferredToSales = 4,
  /** 已完成電銷 */
  CompletedSales = 5,
  /** 已轉業務 */
  TransferredToBusiness = 6,
  /** 商機10%：業務接受 或 booking 接受 */
  Business10Percent = 7,
  /** 商機30% */
  Business30Percent = 8,
  /** 商機70% */
  Business70Percent = 9,
  /** 商機100% */
  Business100Percent = 10,
  /** 商機失敗 */
  BusinessFailed = 11,
  /** 已轉專案 */
  TransferredToProject = 12,
}